internal/kafka/internal/clusters/standalone_provider.go

```diff
@@ -558,3 +558,9 @@ func (s *StandaloneProvider) CreateMachinePool(request *types.MachinePoolRequest
 	// TODO implement
 	return nil, nil
 }
+
+// noop method, it will always return an empty list as a standalone provider does not have any resource quotas
```

Comment:
The description should be updated to match what's being returned

Reply:
updated it there :)